examples/solids/elasticity.h

```diff
@@ -108,6 +108,7 @@ typedef struct AppCtx_private *AppCtx;
 struct AppCtx_private {
   char          ceedResource[PETSC_MAX_PATH_LEN];     // libCEED backend
   char          meshFile[PETSC_MAX_PATH_LEN];         // exodusII mesh file
+  char          outputfolder[PETSC_MAX_PATH_LEN];
```

Comment:
Thanks Leila. 
Just for consistency, you could rename this variable `outputdir` or `outputdirectory` now, but it's really up to you